pkg/fileutils/fileutils_test.go

```diff
@@ -508,7 +509,7 @@ var matchTests = []matchTest{
 	{"*c", "abc", true, nil},
 	{"a*", "a", true, nil},
 	{"a*", "abc", true, nil},
-	{"a*", "ab/c", false, nil},
+	{"a*", "ab/c", true, nil},
```

Comment:
I don't think this change is correct. "a*" means just top level files that start with "a".